src/Chainweb/Version.hs

```diff
@@ -653,8 +650,6 @@ useLegacyCreationTimeForTxValidation
     -> BlockHeight
     -> Bool
 useLegacyCreationTimeForTxValidation Mainnet01 h = h < 449940 -- ~ 2020-04-03T00:00:00Z
-useLegacyCreationTimeForTxValidation Testnet04 h = h < 286231 -- 2020-04-02T02:00:00Z
-useLegacyCreationTimeForTxValidation Development h = h < 150
```

Comment:
Having this in devnet was annoying and slowing down testing. I don't thing we need to cover this in devnet any more.